### ISSUE TITLE
potential issues in tests with sum/precision

### ISSUE THREAD

**Issue by phinate** [anh-tong/signax#23]
hi, just stumbled upon this library from also looking into `signatory` etc -- I was curious about a couple things i saw in your test suite: it looks like you're checking the sum over multiple signatures from `signatory` as opposed to each signature individually, is that by design? i would think that could smear over possible errors there, especially if you keep the error tolerance high like you currently have it.

do the issues with tolerance still appear if you increase the precision of JAX to 64-bit, or is torch also in 32-bit here? i.e. does

```python
from jax.config import config
config.update("jax_enable_x64", True)
```

help at all?

**Comment by anh-tong** [comment]
Thank you for taking a look of this repository and giving us suggestions. 

It was lazy of me to just compare between the sums. Instead, the test should compare individual entries in tensor algebra. At the time writing the code, I was uncertain about the order of flattened vectors in `signatory`.

Your spectaculation on error tollerance is also accurate (I was not thinking of using 64-bit at all). I will try to re-test it with higher precision floating point.

Even though, these can be problematic. Our examples can demonstrate the same result with deep signature transform paper.

**Comment by anh-tong** [comment]
This is now fixed in #26.
